itou/www/invitations_views/tests/tests_prescriber_organization.py

```diff
@@ -1,312 +1,335 @@
+from datetime import timedelta
+
 from allauth.account.models import EmailAddress
 from django.conf import settings
-from django.contrib.auth import get_user_model
-from django.contrib.messages import get_messages
 from django.core import mail
 from django.shortcuts import reverse
 from django.test import TestCase
+from django.utils.html import escape
 
 from itou.invitations.factories import PrescriberWithOrgSentInvitationFactory
 from itou.invitations.models import PrescriberWithOrgInvitation
 from itou.prescribers.factories import PrescriberOrganizationWithMembershipFactory, PrescriberPoleEmploiFactory
 from itou.prescribers.models import PrescriberOrganization
 from itou.siaes.factories import SiaeWithMembershipFactory
 from itou.users.factories import DEFAULT_PASSWORD, JobSeekerFactory, PrescriberFactory, UserFactory
+from itou.users.models import User
 from itou.utils.perms.prescriber import get_current_org_or_404
-from itou.www.invitations_views.forms import NewPrescriberWithOrgInvitationForm
 
 
-#####################################################################
-############################## Views ################################
-#####################################################################
+POST_DATA = {
+    "form-TOTAL_FORMS": "1",
+    "form-INITIAL_FORMS": "0",
+    "form-MIN_NUM_FORMS": "",
+    "form-MAX_NUM_FORMS": "",
+}
+
+INVITATION_URL = reverse("invitations_views:invite_prescriber_with_org")
 
 
 class TestSendPrescriberWithOrgInvitation(TestCase):
     def setUp(self):
-        self.org = PrescriberOrganizationWithMembershipFactory(kind=PrescriberOrganization.Kind.CAP_EMPLOI)
-        self.sender = self.org.members.first()
-        self.guest = UserFactory.build(first_name="Léonie", last_name="Bathiat")
-        self.post_data = {
-            "form-TOTAL_FORMS": "1",
-            "form-INITIAL_FORMS": "0",
-            "form-MIN_NUM_FORMS": "",
-            "form-MAX_NUM_FORMS": "",
-            "form-0-first_name": self.guest.first_name,
-            "form-0-last_name": self.guest.last_name,
-            "form-0-email": self.guest.email,
+        self.organization = PrescriberOrganizationWithMembershipFactory(kind=PrescriberOrganization.Kind.CAP_EMPLOI)
+        self.sender = self.organization.members.first()
+        self.guest_data = {"first_name": "Léonie", "last_name": "Bathiat", "email": "leonie@example.com"}
+        self.post_data = POST_DATA | {
```

Comment:
Found it: Dictionary Merge & Update Operators!!!! J'ai gagné quoi ?

Reply:
Des emojis et de la fierté !